golem-test-framework/src/components/rdb/docker_mysql.rs

```diff
@@ -79,6 +78,15 @@ impl DockerMysqlRdb {
         self.info.public_connection_string()
     }
 
+    pub fn public_connection_string_to_db(&self, db_name: &str) -> String {
```

Comment:
This is just Self#public_connection_string

Reply:
not sure what you mean by that, 
it is not same `public_connection_string` (which is connecting to default database),
this is helper function which allowing to create connection string to different underlying database

Reply:
ah you are right, missed that part